Source/Particles/Collision/ElasticCollisionPerez.H

```diff
@@ -49,7 +49,6 @@ void ElasticCollisionPerez (
     T_R const  dt, T_R const   L, T_R const dV)
 {
 
-    T_R constexpr inv_c2 = T_R(1.0)/(PhysConst::c*PhysConst::c);
```

Comment:
Fixes the warning:
`./Source/Particles/Collision/ElasticCollisionPerez.H:52:19`: warning: unused variable ‘inv_c2’ [-Wunused-variable]
     T_R constexpr inv_c2 = T_R(1.0)/(PhysConst::c*PhysConst::c);